src/components/marker-chart/Canvas.js

```diff
@@ -550,36 +551,90 @@ class MarkerChartCanvasImpl extends React.PureComponent<Props, State> {
       timelineTrackOrganization,
       viewport: { viewportLeft, viewportRight, viewportTop, containerWidth },
     } = this.props;
-    if (x < marginLeft - MARKER_DOT_RADIUS) {
```

Comment:
This was a (very old) mistake: MARKER_DOT_RADIUS is defined as a ratio of rowHeight.